beluga/include/beluga/sensor/data/sparse_value_grid.hpp

```diff
@@ -0,0 +1,99 @@
+// Copyright 2023 Ekumen, Inc.
```

Comment:
@serraramiro1 nit:
```suggestion
// Copyright 2024 Ekumen, Inc.
```
Same elsewhere.

Reply:
Done